eclair-core/src/test/scala/fr/acinq/eclair/testutils/MyCapturingAppender.scala

```diff
@@ -0,0 +1,39 @@
+package fr.acinq.eclair.testutils
+
+import ch.qos.logback.classic.LoggerContext
+import ch.qos.logback.classic.spi.ILoggingEvent
+import ch.qos.logback.core.AppenderBase
+
+/**
+ * Heavily inspired by [[akka.actor.testkit.typed.internal.CapturingAppender]]
+ */
+class MyCapturingAppender extends AppenderBase[ILoggingEvent] {
```

Comment:
Akka's `CapturingAppender` is `private`, unfortunately, so I couldn't call it from `FixtureSpec` to print or clear logs.